### PR TITLE
Update NREL-5MW input files with new mesh motion input section

### PR DESCRIPTION
**Pull request by sayerhs**

### CODE REVIEW

**Comment by sayerhs** [comment]
@ashesh2512: Can you take a look at these diffs so that we can be sure I've not made any silly mistakes?

**Comment by jhux2** [comment]
@sayerhs Do you want me to approve, or would you rather wait for Ashesh?

**Comment by sayerhs** [comment]
> @sayerhs Do you want me to approve, or would you rather wait for Ashesh?

@jhux2 Please review and approve. I won't merge this until @ashesh2512 has had a chance to comment also.

File: nrel_5mw/g0/nrel_5mw_g0_hypre.i [hunk 1]
```diff
@@ -249,25 +254,37 @@ realms:
...
+          - block_204
+          - block_205
+          - block_206
+        motion:
+          - type: rotation
+            omega: 0.0
+
     solution_options:
 
       name: myOptions
```
Comment (jhux2) [inline nrel_5mw/g0/nrel_5mw_g0_hypre.i:285]: Does the name`myOptions` matter?  In other words, would another name be more helpful/relevant?

File: nrel_5mw/g0/nrel_5mw_g0_muelu.i [hunk 2]
```diff
@@ -202,25 +207,37 @@ realms:
...
+          - block_204
+          - block_205
+          - block_206
+        motion:
+          - type: rotation
+            omega: 0.0
+
     solution_options:
 
       name: myOptions
```
Comment (jhux2) [inline nrel_5mw/g0/nrel_5mw_g0_muelu.i:238]: See comment above.

File: nrel_5mw/g1/nrel_5mw_g1_hypre.i [hunk 3]
```diff
@@ -251,26 +245,39 @@ realms:
...
+          - block_205
+          - block_206
+          - block_206.Tetrahedron_4._urpconv
+        motion:
+          - type: rotation
+            omega: 0.0
+
     solution_options:
 
       name: myOptions
```
Comment (jhux2) [inline nrel_5mw/g1/nrel_5mw_g1_hypre.i:278]: See comment above.

File: nrel_5mw/g1/nrel_5mw_g1_muelu.i [hunk 4]
```diff
@@ -203,26 +208,39 @@ realms:
...
+          - block_205
+          - block_206
+          - block_206.Tetrahedron_4._urpconv
+        motion:
+          - type: rotation
+            omega: 0.0
+
     solution_options:
 
       name: myOptions
```
Comment (jhux2) [inline nrel_5mw/g1/nrel_5mw_g1_muelu.i:241]: See comment above.

File: nrel_5mw/g2/nrel_5mw_g2_hypre.i [hunk 5]
```diff
@@ -244,24 +244,39 @@ realms:
...
+          - block_205
+          - block_206
+          - block_206.Tetrahedron_4._urpconv
+        motion:
+          - type: rotation
+            omega: 0.0
+
     solution_options:
 
       name: myOptions
```
Comment (jhux2) [inline nrel_5mw/g2/nrel_5mw_g2_hypre.i:277]: See comment above.

File: nrel_5mw/g2/nrel_5mw_g2_muelu.i [hunk 6]
```diff
@@ -197,24 +197,39 @@ realms:
...
+          - block_205
+          - block_206
+          - block_206.Tetrahedron_4._urpconv
+        motion:
+          - type: rotation
+            omega: 0.0
+
     solution_options:
 
       name: myOptions
```
Comment (jhux2) [inline nrel_5mw/g2/nrel_5mw_g2_muelu.i:230]: See comment above.

**Review by jhux2 — APPROVED** [review]

**Comment by ashesh2512** [comment]
@sayerhs They look fine except the `frame` keyword is missing for all frames where `omega = 0`. It should be set to `non_inertial`.

File: nrel_5mw/g0/nrel_5mw_g0_hypre.i (hunk 1 above)
Comment (sayerhs) [reply nrel_5mw/g0/nrel_5mw_g0_hypre.i:285]: @jhux2 The name can be something descriptive you prefer, it is not used internally for anything.

Comment (jhux2) [reply nrel_5mw/g0/nrel_5mw_g0_hypre.i:285]: It sounds like the name isn't terribly important, so I'm fine keeping it as is.

**Comment by jhux2** [comment]
*[a discussion](https://reviewable.io/reviews/exawind/solver-performance/2#-LX_4f9D9GS0hB3ylOFu:-LX_4f9E25Fd_GqxOs1w:b9u71pn) (no related file):*
Testing whether Reviewable works with PRs.

---

**Pull request merged**